docs/topics/mta-cli-analyze.adoc

```diff
@@ -25,27 +25,96 @@ Usage:
+
+      --https-proxy (string)              HTTPS proxy string URL.
+
+      --incident-selector (string)        An expression to select incidents
+                                          based on custom variables.
+                                          Example:
+                                          !package=io.demo.config-utils
+
+  -i, --input (string)                    Path to application source code or
+                                          a binary.
+
+      --jaeger-endpoint (string)          Jaeger endpoint to collect traces.
+
+      --json-output                       Create analysis and dependency
+                                          output as JSON.
+
+      --list-sources                      List rules for available
+                                          migration sources.
+
+      --list-targets                      List rules for available
+                                          migration targets.
+
+  -l, --label-selector (string)           Run rules based on specified label
+                                          selector expression.
+
+      --maven-settings (string)           Path to the custom maven
+                                          settings file to use.
+
+      --overwrite                         Overwrite output directory.
+
+      --skip-static-report                Do not generate the static report.
+
+  -m, --mode (string)                     Analysis mode, must be one of
+                                          `full` or `source-only`
+                                          (default: `full`).
+
+      --no-proxy (string)                 Proxy-excluded URLs
+                                          (relevant only with proxy).
+
+  -o, --output (string)                   Path to the directory for analysis
+                                          output.
+
+      --overwrite                         Overwrite output directory.
+
+      --rules (stringArray)               Filename or directory containing
+                                          rule files.
+
+      --skip-static-report                Do not generate the static report.
+
+  -s, --source (string)                   Source technology to consider
+                                          for analysis.
+                                          To specify multiple sources,
+                                          repeat the parameter:
+                                          `--source <source_1>
+                                          --source <source_2>` etc.
+
+  -t, --target (string)                   Target technology to consider
+                                          for analysis.
+                                          To specify multiple targets,
+                                          repeat the parameter:
+                                          `S--target <target_1>
+                                          --target <target_2>` etc.
+
```

Comment:
@RichardHoch - My formatting


Preview is [HERE](https://anarnold97.github.io/MTA-Preview/MTA/MTA-3140.html#mta-cli-analyze_cli-guide)